nb/projects/belt_sprockets/belt_sprockets.nb

```diff
@@ -0,0 +1,683 @@
+//Ported to JSxCAD from https://www.thingiverse.com/thing:16627/files
+
+// Parametric Pulley with multiple belt profiles
+// by droftarts January 2012
+
+// Based on pulleys by:
+// http://www.thingiverse.com/thing:11256 by me!
+// https://github.com/prusajr/PrusaMendel by Josef Prusa
+// http://www.thingiverse.com/thing:3104 by GilesBathgate
+// http://www.thingiverse.com/thing:2079 by nophead
+
+// dxf tooth data from http://oem.cadregister.com/asp/PPOW_Entry.asp?company=915217&elementID=07807803/METRIC/URETH/WV0025/F
+// pulley diameter checked and modeled from data at http://www.sdp-si.com/D265/HTML/D265T016.html
+
+// tune-able constants
+
+const makePulley = (profile, teeth) => {
```

Comment:
I didn't realize everything was embedded in this when I first looked at the code.

I think I'd try turning things a bit inside out.

const fooTooth = { profile: ToothProfile(...).view(), spacing: ... };

const FooPulley = teeth => Pulley(fooTooth, teeth).view();

etc.

Which would pull the resources out the top level and make them viewable in the notebook.